src/configs/godot.rs

```diff
@@ -0,0 +1,32 @@
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Deserialize, Serialize)]
+#[cfg_attr(
+    feature = "config-schema",
+    derive(schemars::JsonSchema),
+    schemars(deny_unknown_fields)
+)]
+#[serde(default)]
+pub struct GodotConfig<'a> {
+    pub format: &'a str,
+    pub version_format: &'a str,
+    pub style: &'a str,
+    pub symbol: &'a str,
+    pub disabled: bool,
+    pub show_version: bool,
+    pub version_command: Vec<&'a str>,
+}
+
+impl<'a> Default for GodotConfig<'a> {
+    fn default() -> Self {
+        GodotConfig {
+            format: "via [$symbol($version )]($style)",
+            version_format: "v${raw}",
+            style: "bold blue",
+            symbol: "godot ",
+            disabled: false,
+            show_version: false,
```

Comment:
There is no need for an explicit `show_version` because users can just remove `$version` from the format string instead. If the command is too slow, I would prefer disabling the module by default instead.

Reply:
The reason for not showing the version by default is not because of slowness but because the godot executable is not guaranteed to be in the user's path or have a consistent name.

Would it be acceptable to just remove `$version` from the default format string?

Reply:
For other modules with slow CLIs, we have disabled them by default, so I would prefer doing that for consistency.